WebDriverAgentLib/Categories/XCUIElement+FBUID.m

```diff
@@ -20,12 +21,18 @@ - (NSString *)fb_uid
   if ([self respondsToSelector:@selector(accessibilityElement)]) {
     return [FBElementUtils uidWithAccessibilityElement:[self performSelector:@selector(accessibilityElement)]];
   }
-  // With Xcode 10, using fb_lastSnapshot is faster than resolving and using the lastSnapshot property
-  if (isSDKVersionLessThan(@"13.0")) {
-    return self.fb_lastSnapshot.fb_uid;
+  if ([self.query respondsToSelector:@selector(uniqueMatchingSnapshotWithError:)]) {
```

Comment:
I'd rather flip the if condition and put the immediate return there. the method would be easier to read in such case

Reply:
We could also cache the result into a static variable (taking into account this method is called quite often we migth same some time for `respondsToSelector` verification)